sourmash/commands.py

```diff
@@ -715,13 +715,16 @@ def multigather(args):
     args.db = [item for sublist in args.db for item in sublist]
     inp_files = [item for sublist in args.query for item in sublist]
     if args.query_from_file:
-        more_files = sourmash_args.load_file_list_of_signatures(args.query_from_file)
+        try:
+            more_files = sourmash_args.load_file_list_of_signatures(args.query_from_file)
+        except (UnicodeDecodeError, ValueError) as e:
+            more_files = [args.query_from_file]
         inp_files.extend(more_files)
```

Comment:
If we want to allow lca/index files to be used in `--query-from-file`, this is the critical section. Working for `.sbt.zip` files, but not unzipped json files (e.g. lca test below).

Reply:
ok, so there's two different things going on here -- I think we need to update the documentation to make it clearer, too!

`--query <sigfile or SBT or LCA>` should work

`--query-from-file <text file containing list of files that would normally go in --query>` should also work

so, here, we don't need the try/except or anything. We just need a better error message and/or description of what `--query-from-file` should be.